cloudprem/variables.tf

```diff
@@ -175,7 +175,13 @@ variable "cache_instance_type" {
   default     = "cache.t2.small"
 }
 
-# --- END Databsae and storage Options --- #
+variable "enable_webhooks" {
+  description = "This option will spin up a managed Kafka & Redis cluster to support private webhooks."
+  type        = bool
+  default     = false
+}
```

Comment:
Might be outdated.. this variable to key the webhooks module isn't used quite yet?

Reply:
Used later...